src/app/games/mancala/ba-awa/BaAwaTutorial.ts

```diff
@@ -0,0 +1,88 @@
+import { Tutorial, TutorialStep } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStep';
+import { MGPOptional } from 'src/app/utils/MGPOptional';
+import { BaAwaConfig } from './BaAwaConfig';
+import { BaAwaRules } from './BaAwaRules';
+import { MancalaTutorial } from '../common/MancalaTutorial';
+import { MancalaState } from '../common/MancalaState';
+import { MancalaDistribution, MancalaMove } from '../common/MancalaMove';
+import { TutorialStepFailure } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStepFailure';
+
+const defaultConfig: MGPOptional<BaAwaConfig> = BaAwaRules.get().getDefaultRulesConfig();
+
+export class BaAwaTutorial extends Tutorial {
+
+    public gameName: string = $localize`Ba-awa`;
+
+    public tutorial: TutorialStep[] = [
+        MancalaTutorial.intro(
+            this.gameName,
+            BaAwaRules.get().getInitialState(defaultConfig),
+        ),
+        TutorialStep.informational(
+            $localize`Ba-awa`,
+            $localize`Bonus fact: Ba-awa, also called Adi, is played mainly in Ghana.`,
+            BaAwaRules.get().getInitialState(defaultConfig),
+        ),
+        MancalaTutorial.sowing(
+            new MancalaState([
+                [4, 4, 4, 4, 4, 4],
+                [4, 0, 4, 4, 4, 4],
+            ], 0, [0, 0]),
+        ),
+        TutorialStep.fromMove(
+            $localize`Multiple laps sowing`,
+            $localize`However, the distribution only stops when the last seed is dropped in a house that contains zero or three seeds (before drop).<br/><br/>You're playing Dark, do such a move!`,
```

Comment:
plutôt que 0 ou 3 (avant drop), quid de 1 ou 4 (après y avoir posé votre graine) ?

Reply:
Où alors on explique que ça s'arrête soit sur capture, soit "dans le vide" ?
mais ouais ça me semble clair, à double checker avec des autruis, pour l'ux bien faite

Reply:
ouais, "when the last seed is dropped in an empty house, or when it captures"?